src/nns-testing/src/index.ts

```diff
@@ -193,7 +193,10 @@ async function createRvmProposal(
 
   if (isNullish(config.neuronId)) {
     console.log('No neuron ID found, creating a neuron now...');
-    neuronId = await createNeuron(config);
+    neuronId = await createNeuron({
+      ...config,
+      identity,
+    });
```

Comment:
When there was no identity or neuron, a new identity would be created on line 191, but not passed into this function. So this function would think there is still no identity and it would create a 2nd new identity that would own the neuron. So when the proposal was created, the identity that was used was not the owner of the neuron.